snapshot/snapshot.go

```diff
@@ -539,10 +536,8 @@ func (o *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 			return nil, storage.Snapshot{}, errors.Wrap(err, "failed to stat parent")
 		}
 
+		// FIXME: Why only change owner of having parent?
 		if err := lchown(filepath.Join(td, "fs"), st); err != nil {
```

Comment:
It seems not to change the `err` in func's topmost scope, so I think `cleanupSnapshotDirectory` can't be called properly.